api/services/user_service.py

```diff
@@ -0,0 +1,33 @@
+# This file is treated as service layer
+from api import helpers, models, db
+
+
+class UserService:
+    def create_user(self, data):
+        system_phone_data = models.SystemPhone.query.filter_by(
+            phone=data["to_number"]
+        ).first()
+
+        if not system_phone_data:
+            return
+
+        partner = models.PartnerSystemPhone.query.filter_by(
+            system_phone_id=system_phone_data.id
+        ).first()
+
+        if not partner:
+            return
+
+        user_exists = db.session.query(
+            db.exists().where(models.User.phone.endswith(data["from_number"][-10:]))
+        ).scalar()
```

Comment:
instead of quering in service file we should create a function inside the user model

Reply:
updated